tests/unit/forklift/test_forms.py

```diff
@@ -10,45 +10,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO:
+#  `tests.unit.forklift.test_legacy.TestFileUpload.test_fails_invalid_post_data`
+#  already covers the full form with specific cases.
+#  Should we remove these test cases? We will remain at 100% coverage.
```

Comment:
Answer and resolve this prior to merging.

Reply:
Seems like more test coverage on a critical part of the service is a good thing? I'd vote to keep the form level tests unless there is a good reason to remove them.